src/routes.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Route, Switch, BrowserRouter } from "react-router-dom";
+import Home from "./containers/Home/Home";
+import Login from "./containers/Login/Login";
+import Wrapper from "./hoc/Wrapper/Wrapper";
+import Header from "./components/Header/Header";
+
+const routes = () => {
+  return (
+    <BrowserRouter>
+    <Wrapper>
```

Comment:
what happens when you remove this line

Reply:
Wrapper is an HOC that allows Switch and Header component to sit side by side.

Reply:
<img width="1242" alt="screen shot 2018-08-28 at 14 38 40" src="https://user-images.githubusercontent.com/28563179/44720661-1e1df780-aad0-11e8-95a6-e90040cb0cc9.png">

Reply:
Okay